better_p2p/network_herald.go

```diff
@@ -0,0 +1,155 @@
+package better_p2p
+
+import (
+	"github.com/libsv/go-p2p/chaincfg/chainhash"
+	"github.com/libsv/go-p2p/wire"
+)
+
+type NetworkHerald struct {
+	pm *PeerManager
+}
+
+func NewHerald(m *PeerManager) *NetworkHerald {
+	return &NetworkHerald{pm: m}
+}
+
+func (h *NetworkHerald) CountConnectedPeers() uint {
+	return h.pm.CountConnectedPeers()
+}
+
+// AnnounceTransactions will send an INV messages to the provided peers or to selected peers if peers is nil
+// it will return the peers that the transaction was actually announced to
```

Comment:
```suggestion
// AnnounceTransactions will send an INV messages to the provided peers or to selected peers if peers is nil.
// It will return the peers that the transaction was actually announced to
```